viofs/VirtFS Package/VirtFS Package.vcxproj.filters

```diff
@@ -7,7 +7,7 @@
     </Filter>
   </ItemGroup>
   <ItemGroup>
-    <Inf Include="..\viorng\viorng.inf">
+    <Inf Include="..\pci\viofs.inf">
```

Comment:
I think it is better to squash this commit to the initial commit fo the project.